packages/credentials-interface/src/exchange.types.ts

```diff
@@ -0,0 +1,133 @@
+export type ExchangeInvitation = {
+  type: string;
+  body: {
+    /**
+     * exchange base url
+     */
+    url: string;
+  };
+};
```

Comment:
Sorry @JGiter I know this is the format that I put in the story. However, this was the "to be" format based on on https://energyweb.atlassian.net/browse/REBDEV-89. The actual current format is:
```
"presentationLink": {
    "type": "string",
    "url": "string"
  }
```
Maybe to keep ICL more flexible and avoid needing to change ew-did-registry if the format changes, we can keep `ExchangeInvitation` and `EWF_EXCHANGE_INVITATION` in ICL for now?

Reply:
no problem. from one side its make sense to put exchange types along with exchange logic (in iam-lib), but from the other these types are used in both iam-lib and ssi-hub. Besides it is potentially possible that there will be some relationships between exchange and credentials type

Reply:
Ah good point, maybe best to have the type in ew-did-registry then 👍